internal/commands/status.go

```diff
@@ -38,8 +38,11 @@ func runStatus(dockerCli command.Cli, claimName string, opts credentialOptions)
 		return err
 	}
 	targetContext := getTargetContext(opts.targetContext, dockerCli.CurrentContext())
-
-	driverImpl, err := prepareDriver(dockerCli)
+	bind, err := requiredClaimBindMount(c, targetContext, dockerCli)
+	if err != nil {
+		return err
+	}
+	driverImpl, err := prepareDriver(dockerCli, bind)
```

Comment:
I wonder if there is a way `prepareDriver` could handle the `requiredClaimBindMount` call intenrally (e.g. if it had a few more args) or if this pattern happens enough times to warrant a hlper?

Reply:
the reasoning behind this layout is the "install" command gets the target orchestrator from its opts while the other commands get it from the claim store